local-setup/jupyter/dataloader/crs_loader.py

```
@@ -206,6 +231,273 @@ def create_tenant(self, tenant_code: str, display_name: str = None,
+                    copied += 1
+                elif any(kw in r.text.lower() for kw in ["duplicate", "already exists", "unique"]):
+                    skipped += 1
+                else:
+                    failed += 1
+            except Exception:
+                failed += 1
+
+        print(f"   📋 Schemas: {copied} copied, {skipped} already existed, {failed} failed (of {len(schemas)} total)")
+
+        if failed > 0:
+            print(f"   ⚠️  Some schemas failed but continuing — non-critical schemas may be optional")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)